com.regnosys.rosetta.lib/src/main/java/com/rosetta/model/lib/meta/RosettaMetaData.java

```diff
@@ -15,7 +16,11 @@
 
 	List<Validator<? super T>> dataRules(ValidatorFactory factory);
 	
-	List<Validator<? super T>> choiceRuleValidators();
+	// @Compat. This will be empty, as choice rules are now all data rules.
+	@Deprecated
+	default List<Validator<? super T>> choiceRuleValidators() {
+		return Collections.emptyList();
+	}
```

Comment:
Part of cleanup 1: no need to implement `choiceRuleValidators` anymore. Just return an empty list by default.